aggregate/billing_functions/seqr.py

```diff
@@ -436,14 +437,26 @@ async def generate_proportionate_maps_of_datasets(
 
     logger.info(f'Getting proportionate map for projects: {filtered_projects}')
 
-    analyses, crams = await get_analysis_objects_and_crams_for_seqr_prop_map(
+    # So here
+    project_file_sizes = aapi.get_sample_file_sizes_async(
+        project_names=projects, start_date=str(start.date), end_date=str(end.date)
+    )
+    file_sizes_by_project: dict[str, list[dict]] = {
+        p['project']: p['samples'] for p in project_file_sizes
+    }
+
+    joint_analyses = await get_analysis_objects_and_crams_for_seqr_prop_map(
         start, end, filtered_projects
     )
+
     seqr_map = get_seqr_hosting_prop_map_from(
-        analyses, crams, project_id_map=sm_pid_to_dataset
+        joint_analyses, file_sizes_by_project, project_id_map=sm_pid_to_dataset
     )
     hail_map = get_shared_computation_prop_map(
-        crams, project_id_map=sm_pid_to_dataset, min_datetime=start, max_datetime=end
+        file_sizes_by_project,
+        project_id_map=sm_pid_to_dataset,
+        min_datetime=start,
+        max_datetime=end,
     )
 
     return seqr_map, hail_map
```

Comment:
Should these be re-named from seqr_map and hail_map to something that's more representative? Like shared_cost_map, batch_job_map or something similar?

Reply:
Renamed more consistently :)